README.rst

```diff
@@ -101,171 +85,18 @@ Comment
 | https://smartcity.rbccps.org/api/1.0.0/db                      | https://localhost:10443/api/1.0.0/db                     |
 +----------------------------------------------------------------+----------------------------------------------------------+
 
-For example, registration of device to local middleware is as follows.
```

Comment:
Lets separate out the API calls from main page. This only address register call, we need a detailed page for all API calls in 1.0.0